README.md

```diff
@@ -8,26 +8,26 @@
    <h3 align="center">rp-hal</h3>
 
   <p align="center">
-    A Rust HAL and board support packages for the RP family of microcontrollers from the Raspberry Pi Foundation
+    Rust support for Raspberry Pi Silicon
```

Comment:
Isn't it better to mention that this is about microcontrollers, to avoid confusion with the traditional Raspberry Pi SBCs?

Reply:
Fair point. I'll expand this a little bit.